zebra-consensus/src/block/check.rs

```diff
@@ -193,8 +193,15 @@ pub fn subsidy_is_valid(block: &Block, network: &Network) -> Result<(), BlockErr
         //
         // https://zips.z.cash/protocol/protocol.pdf#fundingstreams
         for (receiver, expected_amount) in funding_streams {
-            let address =
-                subsidy::funding_streams::funding_stream_address(height, network, receiver);
+            if receiver == FundingStreamReceiver::Deferred {
```

Comment:
Consider moving logic from `miner_fees_are_valid()` to `subsidy_is_valid()`, returning the expected total input value of the miner fees, and only checking that the miner fees are more than or exactly the same as expected (depending on the whether NU6 is active and block subsidies must be fully claimed).

Reply:
> this part only runs before the next halving

Whoops, thank you! Fixed in https://github.com/ZcashFoundation/zebra/pull/8694/commits/8f00f227556f1229141c20e34284a8c3af4c6213 (along with a minor refactor/simplification, let me know if it would be better to minimize the changes and only update that one condition)